docusaurus/docs/dev-docs/intro.md

```diff
@@ -48,30 +57,5 @@ Clicking on any of the following cards will direct you to the introduction page
 :::tip Tips
 - If you already know exactly what you're searching for, use the search bar at the top or navigate using the table of content.
 - If you prefer learning more about Strapi while looking at the project code structure, you can use the interactive [project structure](/dev-docs/project-structure) page.
+- If demos are more your thing, we have a [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo).
```

Comment:
```suggestion
- If demos are more your thing, feel free to watch the [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo).
```
I don't know if it's still a thing but at some point we were encouraged to not use "we" too much so just in case it's still relevant, I suggest this little modification!